prep-javascript-objects/main.js

```diff
@@ -0,0 +1,9 @@
+var person = { firstName: 'Carl', lastName: 'Garth', hobby: 'running' };
+console.log(person);
+var fullName = 'Carl Garth';
+console.log("The person's name is:" + ' ' + fullName + '.');
+person.job = 'chef';
```

Comment:
Hi Nadia, you can use dot notation to access the `firstName` and `lastName` properties of the person object

```js
var fullName = person.firstName + ' ' + person.lastName;
```